package.json

```diff
@@ -2,17 +2,28 @@
   "name": "react-directus",
   "version": "0.0.2",
   "description": "A set of React components and utilities for Directus Headless CMS",
+  "homepage": "https://github.com/gremo/react-directus",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gremo/react-directus.git"
+  },
+  "type": "module",
```

Comment:
I think this is also related to rollup, we can drop `bundleConfigAsCjs`

Reply:
Yes, you are right.

Reply:
Ok i think we could leave rollup config for a future optimiziation and put type:module here as you suggested.